introduction.tex

```diff
@@ -170,6 +170,9 @@ \section{Supported Features}
        involving any hart. (Optional)
    \item A RISC-V hart can be halted when a trigger matches the PC,
        read/write address/data, or an instruction opcode. (Optional)
+    \item Harts can be grouped, and harts in the same group will all halt when
+        any of them halts. These groups can also react to or notify external
```

Comment:
"notify" external triggers? Or "raise"/"cause" external triggers? I don't know what it means to "notify" a trigger.

Reply:
How about "react to or assert external signals"? (Together with rewriting other uses of trigger in this context to signal.) Trigger inherently implies an input.